src/SDK/Trace/Behavior/SpanExporterTrait.php

```diff
@@ -44,7 +56,87 @@ public function export(iterable $spans): int
             return SpanExporterInterface::STATUS_SUCCESS;
         }
 
-        return $this->doExport($spans); /** @phpstan-ignore-line */
+        // If retryPolicy is not set then just doExport once and return the status
+        if ($this->retryPolicy === null) {
+            self::logInfo('retry polict is null. Exporting the spans without retry' . PHP_EOL);
```

Comment:
Having an info log message for this on every export seems superfluous.

Reply:
ok I will change it to debug level